tests/Doctrine/Tests/ORM/Functional/Ticket/DDC1238Test.php

```diff
@@ -56,7 +56,7 @@ public function testIssueProxyClear(): void
 
         $user2 = $this->_em->getReference(DDC1238User::class, $userId);
 
-        $user->__load();
```

Comment:
@beberlei Do you still remember what you intended to do here? Why should it be possible or allowed to load an entity through a reference when the entity manager has been `clear()`ed in the meantime?

Reply:
I believe this tests that the proxy generation has a shortcut for `getId` methods that returns the value passed to getReference(.., $id);

I am not sure this needs the calls to __load anymore, but really don't know anymore.

Reply:
Do you think it should be possible to initialize a proxy _after_ the EM it was obtained from has been cleared?

I don’t think so.

Reply:
maybe I'm missing some part of the conversation, but isn't this the same as initializing the proxy before anything was put into the EM? I mean looking at the three lines of code shown in Gh, it wasn't cleared between `getReference` and `__load`, so it should be fine, no?